### DISCUSSION TITLE
Setting environment variables from package

### DISCUSSION THREAD

**Discussion by amklinv-nnl** [GoogleCloudPlatform/ramble#309] [General]
I would like to add a benchmark that uses an environment variable to locate an input file. I know users can specify environment variables in the configuration files, but I would like to set it in the package itself (since ramble will be downloading the input file). Do you have documentation on how to do this?

**Comment by douglasjacobsen** [comment]
You can't actually do that yet in a nice way, though we have a task to implement that.

I think the closest thing you could do for now would be something like this:

```
executable('set-path-env-var', 'export MY_VAR={my_value}', redirect='', output_capture='')
```

(See: https://googlecloudplatform.github.io/ramble/ramble.language.html#ramble.language.application_language.executable)

The only problem with this is it's not shell agnostic like an explicit environment variable directive would be. But it at least would get you something that works today.
